tensorflow/python/ops/special_math_ops.py

```diff
@@ -134,64 +137,57 @@ def bessel_i1(x, name=None):
 @tf_export('einsum', 'linalg.einsum')
 def einsum(equation, *inputs, **kwargs):
   """A generalized contraction between tensors of arbitrary dimension.
-
   This function returns a tensor whose elements are defined by `equation`,
   which is written in a shorthand form inspired by the Einstein summation
   convention.  As an example, consider multiplying two matrices
   A and B to form a matrix C.  The elements of C are given by:
-
   ```
     C[i,k] = sum_j A[i,j] * B[j,k]
   ```
-
   The corresponding `equation` is:
-
```

Comment:
Does this still render correctly without the empty lines?

@MarkDaoust do you know without testing?

Reply:
>Does this still render correctly without the empty lines?

It won't, please leave them in.

Reply:
Thank you for pointing this out. Apparently, some copy/paste-step caused them to vanish..